test-resources/kind-config.yaml

```diff
@@ -1,6 +1,8 @@
-# See CONTRIBUTING.md -> "Semi-Manual Test Scenarios" and "Running End-to-End Tests on kind" for instructions to run
-# semi-manual and e2e tests on a kind cluster.
-
+# See CONTRIBUTING.md -> "Setting Up a Kind Cluster for Local Testing" for instructions.
```

Comment:
This is actually the file previously known as test-resources/kind-config-lr.yaml.